src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioHostContextService.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation
+{
+    [ExportWorkspaceService(typeof(IHostContextService), ServiceLayer.Host), Shared]
+    internal class VisualStudioHostContextService : IHostContextService
+    {
+        public bool SolutionExistsAndFullyLoaded
+            => KnownUIContexts.SolutionExistsAndFullyLoadedContext.IsActive;
```

Comment:
I kind of like things working in misc files while debugging.  Maybe we could instead take a look at the state flags in `VisualStudioProjectTracker` and only disable this during solution load, but leave it enabled before and after?